libbeat/docs/repositories.asciidoc

```diff
@@ -121,6 +121,7 @@ sudo apt-get update && sudo apt-get install {beatname_pkg}
 ["source","sh",subs="attributes"]
 --------------------------------------------------
 sudo update-rc.d {beatname_pkg} defaults 95 10
+sudo systemctl enable heartbeat-elastic
```

Comment:
You don't need the other command at all on newer OSes with systemd. I think it would make sense to put the `systemctl` command first in the docs. Then have a sentence afterwards that says if your system does not use systemd then run `sudo update-rc.d {beatname_pkg} defaults 95 10`. It's the same with `chkconfig` on RHEL.